src/routes/components/AllComponents.tsx

```diff
@@ -254,7 +330,10 @@ export default function CheckboxPage() {
 
   function getComponentsByFilter(): ReactNode[] {
     return cards
-      .filter(card => card.name.includes(filter) || card.tags?.includes(filter))
+      .filter(
+        card =>
+          card.name.includes(filter.toLowerCase()) || card.tags?.includes(filter.toLowerCase())
```

Comment:
Since some of the tags are rather long maybe a partial match on the tag should be allowed.

No one will search for "structure and navigation", but they might search for "nav".